src/Meshes/interval.jl

```diff
@@ -1,3 +1,5 @@
+using Roots
```

Comment:
Not a big issue, but should we use `RootSolvers` instead?

Reply:
Do you recommend RootSolvers?